src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -387,4 +401,26 @@ TEST_CASE("request_rate_sequence")
   trrm.TestSequences();
 }
 
+TEST_CASE("request_rate_streaming: test that streaming-specific logic works")
```

Comment:
What is happening here? Are you manually creating a single 'thread' the same way the code would?

Reply:
Yes, and running the `Infer()` function in the background and quickly ending it by setting `early_exit = true` then ending the thread.